api/lib/domain/usecases/get-attendance-sheet.js

```diff
@@ -26,11 +27,14 @@ module.exports = async function getAttendanceSheet({
   const isEndTestScreenRemovalEnabled = await endTestScreenRemovalService.isEndTestScreenRemovalEnabledBySessionId(
     sessionId
   );
+  const hasSupervisorAccess = await supervisorAccessRepository.sessionHasSupervisorAccess({ sessionId });
+  const addEndTestScreenColumn = !(isEndTestScreenRemovalEnabled && hasSupervisorAccess);
```

Comment:
c'est psychologique mais je trouve ca plus clair dans l'autre sens
```suggestion
  const addEndTestScreenColumn = !isEndTestScreenRemovalEnabled || !hasSupervisorAccess);
```

Reply:
Done